webpack.config.js

```diff
@@ -2,12 +2,12 @@
 
 const path = require('path');
 
-module.exports = {
-  mode: 'production',
+const umdConfig = {
```

Comment:
I would reuse some common parts of the configuration in the following way:

```
const commonConfig = {
  entry: path.resolve(__dirname, 'src/workast.js'),
  module: {
    rules: [
      {
        test: /\.js$/,
        include: [
          path.resolve(__dirname, 'src')
        ],
        use: {
          loader: 'babel-loader'
        }
      }
    ]
  }
};
```

And then just use `Object.assign` to create each specific configuration. I guess we can get rid of the `browserConfig` and just stick with the minified one which has source maps.

On the other hand, I think we need a more scalable way to add packages to `package.json` and not need to update the externals all the time. Take a look at https://www.npmjs.com/package/webpack-node-externals since it seems it is the solution we are looking for.

Reply:
Thanks for the feedback, I agree with everything i will take care of the necessary changes late today and let you know when it's ready for review again.